src/map/skill.cpp

```diff
@@ -7630,6 +7630,22 @@ int skill_castend_nodamage_id (struct block_list *src, struct block_list *bl, ui
 
 		break;
 
+	case NPC_IMMUNE_PROPERTY:
+		switch (skill_lv) {
```

Comment:
Just a general question:
Are we able to confirm the connection between level and SC somehow?
Just so that we do not get into trouble later on when taking over skill levels for monsters.

For example MD_EVENT_HEMEL (Ele: Holy) uses level 8 (Ele: Poison). Based on the attribute table this is only 25%. I rather think 8 will be Dark or Undead.

Another example OSIRIS (Ele: Undead) uses level 9 (Ele: Ghost). Based on the attribute table this is 175%. So this looks plausible at least. Even if Holy or Fire would make more sense.

I got the following list from @Dia with skill usages over level 5, if it helps somehow:
```
1038
1492
20196
20230
20235
20572
20573
20812
21338
21340
```

Reply:
I've tested in KRo for several MVPs using the skill, and yes there's some incomprehensible element selection but I think this is an official intension.

For example of DARK_SNAKE_LORD(ID:1418), his skill immunes nuetral element(level 1), though he would never been damaged by neutral because his element is a ghost. At first I thought this is an error of DP but as I tested in KRo he does immune neutral. He can be damaged by neutral when he use `NPC_CHANGEDARKNESS` or `NPC_CHANGEHOLY` but actually if neutral element was selected consciously of these, it should be better to immune dark or holy element.

So the conclusion is, KRo doesn't really considered about how effective the immuning element is.

Reply:
![image](https://user-images.githubusercontent.com/47050704/224586966-1ddbb79d-48d4-4c84-9e6f-63276caf9421.png)
Comfirmed 7lv skill holy.
Chinese character above mob shows what elements are being immuned